adapter/go.mod

```diff
@@ -60,7 +60,7 @@ require (
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	go.mongodb.org/mongo-driver v1.7.5 // indirect
-	golang.org/x/crypto v0.0.0-20220511200225-c6db032c6c88 // indirect
+	golang.org/x/crypto v0.1.0 // indirect
```

Comment:
shall we do a go mod tidy and see if there are any go.sum changes to move in.

Reply:
Fixed via https://github.com/wso2/product-microgateway/pull/3303/commits/937f11db1f7192565fbc3ef9ff37db9dbe0c7d36